### DISCUSSION TITLE
Nigeria Odysee link

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#461] [Show and tell]
https://cloud.odysee.live/content/0edfb4b7fb52d2d5ae30e052ce6b61d376fcd662/master.m3u8

**Comment by kaikyoofers** [comment]
Working link: https://cdn.odysee.live/content/0edfb4b7fb52d2d5ae30e052ce6b61d376fcd662/master.m3u8
Also, where did you get this? I want to add it but i dont know the channels name due to it being in a different language. @BellezaEmporium

**Reply by BellezaEmporium** [reply]
I do not remember where I found it. It's indeed one rare find.
